ecosystem.html

```diff
@@ -0,0 +1,1629 @@
+                >Free trial</a
+              >
+            </li>
+          </ul>
+        </div>
+      </div>
+    </nav>
+    <!-- END NAVIGATION -->
+
+    <!-- BEGIN PAGE TITLE -->
+    <header class="headline py-5">
+      <div class="container">
+        <div class="row justify-content-center">
+          <div class="col text-center">
+            <svg
+              version="1.1"
+              xmlns="http://www.w3.org/2000/svg"
+              xmlns:xlink="http://www.w3.org/1999/xlink"
+              x="0px"
+              y="0px"
+              width="98.5px"
+              height="98.5px"
+              viewBox="0 0 98.5 98.5"
+              style="overflow: visible; enable-background: new 0 0 98.5 98.5"
+              xml:space="preserve"
+              class="mb-3"
+            >
+              <style type="text/css">
+                .ansLogo_st0 {
+                  fill: #5bbdbf;
+                }
+                .ansLogo_st1 {
+                  fill: #ffffff;
+                }
+              </style>
+              <defs></defs>
+              <g
+                id="Ansible-Mark-RGB_Pool_1_"
+                transform="translate(-46.207 -46.127)"
+              >
+                <path
+                  id="Path_13_1_"
+                  class="ansLogo_st0"
+                  d="M144.7,95.4c0,27.2-22,49.2-49.2,49.2s-49.2-22-49.2-49.2s22-49.2,49.2-49.2l0,0
+                    C122.6,46.1,144.7,68.2,144.7,95.4"
+                />
+                <path
+                  id="Path_14_1_"
+                  class="ansLogo_st1"
+                  d="M96.3,76.9l12.7,31.4L89.8,93.2L96.3,76.9z M118.9,115.6L99.3,68.4c-0.5-1.3-1.7-2.1-3-2.1
+                    c-1.4,0-2.6,0.8-3.1,2.1l-21.5,51.7H79l8.5-21.3l25.4,20.5c0.7,0.7,1.7,1.2,2.7,1.2c1.9,0,3.6-1.5,3.6-3.4c0,0,0-0.1,0-0.1
+                    C119.2,116.5,119.1,116.1,118.9,115.6"
+                />
+              </g>
+            </svg>
+
+            <h1 class="mb-2">Ansible Ecosystem</h1>
+            <p class="subhead">
+              Find documentation for projects across the Ansible ecosystem.
+            </p>
+          </div>
+        </div>
+      </div>
+    </header>
+    <!-- END PAGE TITLE -->
+
+    <!-- BEGIN MAIN CARDS -->
+    <section class="cards pt-0">
+      <div class="container">
+        <div class="row">
+          <div class="col-md-6 col-lg-4">
+            <a href="core.html">
+              <div class="card border">
+                <div class="card-body">
+                  <h4 class="card-title">Ansible Core</h4>
+                  <p class="card-text">
+                    Ansible Core is the language and runtime that powers
+                    automation.
+                  </p>
+                </div>
+                <div class="card-footer">
+                  <a href="core.html" class="btn btn-outline-black"
+                    >Visit the documentation</a
```

Comment:
This is a bit misleading since it goes to yet another html page. This happens on the main index page as well. We should save 'Visit the documentation' for links that actually go straight to the docs.  On the ones that go to sub html pages, there's still the 'learn more' label for the links that actually do go straight to docs I think.